src/main/java/Duke.java

```diff
@@ -1,10 +1,59 @@
+import Command.Command;
```

Comment:
This file has incorrect indentation level, plus multiple lines of unnecessary newlines.